docs/initializing.rst

```diff
@@ -555,6 +556,42 @@ point into the system. It will handle the setup and initialization of the config
 and start up a Toolkit session once all the required pieces have been initialized and set up.
 
 
+Distributing application code from your site
+============================================
+
+If you want to distribute custom application code that may only be accessible for
+users on your local network or from developers who have access keys to the Git
+repositories, you can upload individual bundles to Shotgun and then take over the ``populate_bundle_cache_entry`` and ``can_cache_bundle`` methods of the
+``core/bootstrap.py`` hook.
```

Comment:
i am missing the word 'cache' in this sentence. maybe we can add that shotgun can act as a cache layer, you can cache any descriptor type, even app store descriptors, and as soon as you have access to Sg, you have access to this cache?

Reply:
i guess i hadn't thought about it from a performance point of view, more of an accessibility point of view. Sorry for not being clear. I think what i was trying to say was that we should express the fact that this is a _fallback_ - you can basically set up a cloud based fallback where you setup goes to look before it looks in git, app store etc. Does that make more sense?